tools/DevCheck.ps1

```diff
@@ -255,14 +255,15 @@ function Test-DevTestCert
     $pfx = Get-PfxCertificate -FilePath $path
     $thumbprint = $pfx.Thumbprint
 
-    $cert = Get-ChildItem -Path "cert:\LocalMachine\TrustedPeople\$thumbprint"
```

Comment:
Another pattern for this is:

```ps1
$cert = Get-ChildItem -Path "cert:\LocalMachine\TrustedPeople\$thumbprint" -ErrorAction SilentlyContinue;
if ($cert)
{
    // use cert
}
else
{
    // complain
}
```

Avoids two tests, lots of red, and a "not foo"

Reply:
-ErrorAction is cute. Thanks for the tip. Will look into it [when next in here](https://github.com/microsoft/ProjectReunion/issues?q=is%3Aissue+is%3Aopen+devcheck)